desktop/hardened-desktop/admin-controls/index.md

```diff
@@ -0,0 +1,57 @@
+---
+description: admin controls for desktop
+keywords: admin controls, rootless, docker desktop, hardened desktop
+title: What is Admin Controls?
+--- 
+>Note
+>
+>Admin Controls is currently in [Early Access](../../../release-lifecycle.md#early-access-ea) and available to Docker Business customers only. 
```

Comment:
Let's remove any mention of Early Access throughout the docs. We'll be immediately releasing all this functionality as GA.